<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoSCurveDialog</class>
 <widget class="QDialog" name="AutoSCurveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="scurve_generation">
      <attribute name="title">
       <string>  S Curve Generator Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_inst">
           <property name="text">
            <string>Instrument</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_inst">
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_mat">
           <property name="text">
            <string>Source material, units</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_mat">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_matdose">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_back">
           <property name="text">
            <string>Define static scenario backgrounds</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_bgnd">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Background Material Entry</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="check_invert">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Invert S-Curve</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_24">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_dwell">
             <property name="text">
              <string>Dwell Time (s): </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_dwell">
             <property name="text">
              <string>30</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_rep">
             <property name="text">
              <string>Repetitions: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_rep">
             <property name="text">
              <string>100</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_resulttype">
             <property name="text">
              <string>Fitting S-curve for:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="combo_resulttype">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>PID</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>C&amp;C</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>TP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Precision</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Recall</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Fscore</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advanced_options">
      <attribute name="title">
       <string>Advanced Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_19">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of points on the edge of the S-curve going from 0 to 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Points on Rising Edge</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="line_edge">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_50">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QCheckBox" name="check_minx">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If you know the range that the rising edge might be, guessing near it might hasten convergence&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Min x init guess</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="line_minx">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>0.00000001</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QCheckBox" name="check_maxx">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If you know the range that the rising edge might
                 be, guessing near it might hasten convergence&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                </string>
               </property>
               <property name="text">
                <string>Max x init guess</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="line_maxx">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>0.001</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Using less repetitions may improve convergence speed, but has a higher risk of giving erroneous results&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Repetitions for initial guesses</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="line_initrep">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QCheckBox" name="check_addpoints">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;These points will help find the S-curve and will always be included in the final S-curve plot and results&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Additional points to include in S-Curve</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_addpoints">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_68">
         <item>
          <widget class="QCheckBox" name="check_cleanup">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Remove S-curve range-finding scenarios from database upon convergence</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_67">
         <item>
          <widget class="QCheckBox" name="check_name">
           <property name="text">
            <string>Set custom group name for S-curve scenarios</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_name">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>[Default]</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_21">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Highest value that will be considered as the bottom of the S-curve. Must be between 0 and 1.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Lower bound for S-Curve:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_lowerbound">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>0.1</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="label_22">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Lowest value that will be considered as the top of the S-curve. Must be between 0 and 1.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Upper bound for S-Curve:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_upperbound">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>0.9</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AutoSCurveDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AutoSCurveDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_name</sender>
   <signal>clicked(bool)</signal>
   <receiver>line_name</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combo_mat</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>combo_matdose</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_minx</sender>
   <signal>clicked(bool)</signal>
   <receiver>line_minx</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_maxx</sender>
   <signal>clicked(bool)</signal>
   <receiver>line_maxx</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>673</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_addpoints</sender>
   <signal>clicked(bool)</signal>
   <receiver>line_addpoints</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
